tests/jsx-split-attrs/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -90,11 +90,21 @@ long_open_long_children = (
       Hello world
     </BaseForm>
     <div>
-      <div><div><div><div><div>hey hiya how are ya</div></div></div></div></div>
+      <div>
+        <div>
+          <div>
+            <div>
+              <div>hey hiya how are ya</div>
+            </div>
+          </div>
+        </div>
+      </div>
```

Comment:
One consequence of the new heuristic is that deeply nested tags now expand to take up more lines. This might be nice as it makes it very explicit that you have a high level of nesting.

If we want to avoid this we could change the heuristic to only expand elements if they have at least two JSX tags.